src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -79,7 +79,7 @@ public static CodeWriter WriteFieldDeclaration(this CodeWriter writer, FieldDecl
 
             if (field.WriteAsProperty)
             {
-                writer.AppendRaw(modifiers.HasFlag(FieldModifiers.ReadOnly) ? "{ get; }" : "{ get; set; }");
+                writer.LineRaw(modifiers.HasFlag(FieldModifiers.ReadOnly) ? "{ get; }" : "{ get; set; }");
```

Comment:
I'm not sure about this one. You probably want to change the line below:
```cs
return field.WriteAsProperty ? writer.Line() : writer.Line($";");
```

Reply:
It's the same as AppendRaw() + Line(). So result will be the same, either we do LineRaw() here or we do AppendRaw() and add an empty line later.

Reply:
Yes, but before adding a new line, we can assign a default value, e.g.:
```cs
public string MyProp { get; set; } = "abc";
```